spec/support/time_helpers.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
```

Comment:
Following on from a previous comment we are currently using [Timecop](https://github.com/travisjeffery/timecop) for any date stuff in the tests.

However what do you know, you can do this with just rspec! 🤦‍♂️ 

So I think we should move to using the rspec timehelpers, but I think it should be done as a single task rather than have a mix of ways of playing with time in the tests. So till then I'd be looking to remove this and stick to using Timecop to keep things consistent.

Reply:
I have seen Timecop in the gemspec, but I couldn't find any single test that uses Timecop in this codebase. Are you sure it is actually used? Because if not, we can just remove the gem and keep this support file :D

Reply:
On a side note, this support is not coming from RSpec but from Rails. I think it was introduced in Rails 4.2. It definitely wasn't' supported by Rails 3 :D

Reply:
Think you are right, was probably me chucking it in as a copy and paste job from Waste Carriers.

However I'd still like to stick with Timecop for now, because when I talk about consistency I mean across both services. And when we switch to using the rspec helpers it would be across both as well.

I just don't like the idea of having one style in WCR, and another one in WEX. However @irisfaraway does most of the work anyways, so if she feels differently I'm happy to pipe down 😁

Reply:
I'm happy either way too :) is an easy transition anyway as the syntax is basically the same :D So is an @irisfaraway decision ;)

Reply:
For now, I have used Timecop, will be easy to bring back the Time helper support stuff :) let's discuss it on Tuesday